dvc/command/add.py

```diff
@@ -14,49 +15,51 @@ class CmdAdd(CmdBase):
     def run(self):
         try:
             if len(self.args.targets) > 1 and self.args.file:
-                raise DvcException("can't use '--file' with multiple targets")
+                raise RecursiveAddingWhileUsingFilename()
 
-            for target in self.args.targets:
-                self.repo.add(
-                    target,
-                    recursive=self.args.recursive,
-                    no_commit=self.args.no_commit,
-                    fname=self.args.file,
-                )
+            self.repo.add(
+                self.args.targets,
+                recursive=self.args.recursive,
+                no_commit=self.args.no_commit,
+                fname=self.args.file,
+                progress=True,
```

Comment:
I don't think we need this one. Only disabling the progress bar won't help anyone much, as he will be bombarded by our other messages anyway. We will need to come up with a silencing strategy for dvc in general later. (could do that through logger right now, though).
```suggestion
```

Reply:
@casperdcl `progress` is still here, why is this one marked as resolved?

Reply:
resolved locally, haven't pushed yet because it's related to a comment (below). Not sure if it went through the github interface, though. trying to recomment...

Reply:
Ah, got it. Sorry :)

Reply:
yes I think I hit Ctrl+Enter which started a (pending) review rather than adding a single comment so my message didn't go through.